product-tours/tour-tf-keras/Tour-With-TensorFlow-and-Keras.ipynb

```diff
@@ -0,0 +1,2701 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "QXGSKS1u0n8x",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "# Tour With TensorFlow and Keras"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Install dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "DO7yawFVxVIt",
+    "tags": [
+     "installation"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install neptune-client==0.4.129 neptune-contrib==0.24.9 tensorflow==2.3.1 scikit-plot==0.3.7 matplotlib==3.3.3"
```

Comment:
Some of those installs (matploltib, scikit-plot) are not needed in the basic version. 
They should be moved to the advanced section. 

Remember that we probably have to freeze the matplotlib/pyplot versions so in the updated_libs you would actually not have anything :(

Reply:
moved some installs to adv.
We do not use log chart here -> no need to freeze dependencies.